Projects/TEAM-3/Mykhailenko/QA2022_04_320/ex320/UI.cs

```diff
@@ -0,0 +1,246 @@
+            _f.Add((Figure)(new Rectangle("Rectangle7", w_color.red, new DateTime(2022, 04, 18), 7, 9)));
+            _f.Add((Figure)(new Trapezoid("Trapezoid1", w_color.white, new DateTime(2022, 04, 18), 10, 5, 4)));
+            _f.Add((Figure)(new Trapezoid("Trapezoid2", w_color.grey, new DateTime(2022, 04, 20), 12, 7, 5)));
+            _f.Add((Figure)(new Trapezoid("Trapezoid3", w_color.silver, new DateTime(2022, 04, 17), 7, 5, 4)));
+            _f.Add((Figure)(new Trapezoid("Trapezoid4", w_color.black, new DateTime(2022, 04, 19), 15, 7, 5)));
+            Console.WriteLine("Figures added to the list\n");
+        }
+
+        public static void AddAnyFigure(List<Figure> _fl)
+        {
+            int ask = 0;
+            string type_f="";
+
+            Console.WriteLine("Choose the type of shape you want to add");
+            Console.WriteLine("1. Circle");
+            Console.WriteLine("2. Rectangle");
+            Console.WriteLine("3. Trapezoid");
+            try { ask = int.Parse(Console.ReadLine()); }
+            catch (Exception ex) { ask = 0; }
+            switch (ask)
+            {
+                case 1: 
+                    {
+                        Figure f = new Circle();
+                        type_f = "Circle";
+                        f.Input();
+                        _fl.Add(f);
+                        break;
+                    }
+                case 2:
+                    {
+                        Figure f = new Rectangle();
+                        type_f = "Rectangle";
+                        f.Input();
+                        _fl.Add(f);
+                        break;
+                    }
+                case 3:
+                    {
+                        Figure f = new Trapezoid();
+                        type_f = "Trapezoid";
+                        f.Input();
+                        _fl.Add(f);
+                        break;
+                    }
+                default: break;
+            }
+            Figure tmp = _fl[_fl.Count - 1];
+            Console.WriteLine("\nShape ({0}) added to the end of the list\n",type_f);
+            tmp.Output();
+            Console.WriteLine();
+        }
+
+        public static void DeleteAnyFigure (List<Figure> _fl)
+        {
+            int d = 0;
+
+            Console.Write("Select the number of the shape list you want to delete : ");
+            try { d = int.Parse(Console.ReadLine()); }
+            catch (Exception ex) { d = 0; }
+            if (d > 0 && d <= _fl.Count)
+            {
+                Figure tmp = _fl[d - 1];
+                Console.WriteLine("The shape {0} is deleted", isName(tmp));
+                tmp.Output();
+                Console.WriteLine();
+                _fl.RemoveAt(d - 1);
+            }
+            else { Console.WriteLine("There is no such item in this list.\n"); }
+        }
+
+        public static void Sort (List<Figure> _fl) //sortered rectangles add first in list of figures
```

Comment:
It could be better to sort with LINQ
https://docs.microsoft.com/ru-ru/dotnet/csharp/linq/